### DISCUSSION TITLE
<towxml>在uniapp的<scroll-view>内无法滚动？

### DISCUSSION THREAD

**Discussion by JalenChung** [sbfkcel/towxml#294] [Q&A]
正常来说<scroll-view>内设置一个动态的scroll-into-view值，就可以滚动到元素的相应位置（就像h5的a标签锚点），但在uniapp的<scroll-view>内无法滚动？如果我不用towxml的话程序是没有问题的。以下是我的元素结构：
![image](https://github.com/user-attachments/assets/434a33e3-f4f0-4895-9456-4e258fb40676)

**Comment by sbfkcel** [comment]
检查样式是否冲突
